### ISSUE TITLE
-Bwesn alone does nothing

### ISSUE THREAD

**Issue by joa-quim** [GenericMappingTools/gmt#134]
Not sure this should not classify as a bug. Let's say I want to plot the four axes but no annotations, not ticks, than this should do it

```
psbasemap -JX12c/0 -Bwesn -R1/10/1/10 -P > lixo.ps
```
but not. It plots nothing. To plot the axes only and no fiddling with the tick lengths I have to cheat it a bit, asking for annotations that are outside the -R range.
```
psbasemap -JX12c/0 -Ba100 -Bwesn -R1/10/1/10 -P > lixo.ps
```

**Comment by joa-quim** [comment]
OK, but ``-Bwesn`` is more intuitive (``-B0`` is shortcut that one must remember that it exists). Both should work.

**Comment by seisman** [comment]
Agree with @joa-quim.

**Comment by seisman** [comment]
Fixed in #137.
